cluster-stamp.bicep

```
@@ -192,6 +160,29 @@ resource pipPrimaryCluster 'Microsoft.Network/publicIPAddresses@2022-05-01' exis
   name: 'pip-BU0001A0005-00'
 }
 
+@description('The in-cluster ingress controller identity used by the pod identity agent to acquire access tokens to read SSL certs from Azure Key Vault.')
+resource miIngressController 'Microsoft.ManagedIdentity/userAssignedIdentities@2022-01-31-preview' existing = {
+  name: 'mi-${clusterName}-ingresscontroller'
```

To be consistent with your other references

```suggestion
  scope: resourceGroup()
  name: 'mi-${clusterName}-ingresscontroller'
```